### DISCUSSION TITLE
Configure SSL/TLS Protocol Settings

### DISCUSSION THREAD

**Discussion by kbcitrite** [ipmitool/ipmitool#271] [Q&A] [answered]
Hello,

I'm looking for a way to disable TLS v1.0 on a Citrix MPX appliance's ILO interface and can't find a way to do it via the GUI or CLI (I've reached out to the vendor as well). 

Does anyone know if ipmitool can do this? I saw cipher suites in `lan set`, but didn't see anything obvious about enabling/disabling of TLS versions:

![image](https://user-images.githubusercontent.com/30004521/108900800-116ab380-75e8-11eb-9ec2-6f19ff6e424f.png)

Thanks!
-Kenny

**Comment by doublechiang** [comment, marked as answer]
It is controlled by that appliance. Ask your vendor as you said.

**Reply by kbcitrite** [reply]
Great, thank you for confirming!
